cfgov/jinja2/v1/_includes/organisms/secondary-navigation.html

```diff
@@ -40,24 +40,28 @@
      aria-label="Section navigation">
     {% set sec_nav_settings = {
         'label': 'In this section',
-        'is_bordered': false,
         'is_midtone': true,
+        'is_bordered': false,
         'is_expanded': false
     } %}
 
     {% from 'organisms/expandable.html' import expandable with context %}
     {% call() expandable(sec_nav_settings) %}
         {% import 'molecules/nav-link.html' as nav_link %}
-        <ul class="o-secondary-navigation_list
+        <ul class="m-list
+                   m-list__unstyled
+                   o-secondary-navigation_list
                    o-secondary-navigation_list__parents">
         {%- for item in nav_items %}
-            <li data-nav-is-active="{{ item.expanded }}">
+            <li class="m-list_item" data-nav-is-active="{{ item.expanded }}">
```

Comment:
Any idea how/where `data-nav-is-active` is used?

Reply:
https://github.com/cfpb/cfgov-refresh/pull/4365/files#diff-ea7aba4c4e268a13b784481eca86405dR75

Reply:
No, it seems to be specific to the secnav though